### PR TITLE
Create ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60355](https://cumulocity.atlassian.net/browse/MTM-60355)
Original PR: [6883](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6883)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: otpAuth from chl or data (#6869) [GRAFT][release/cd] (#6883)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Support otpAuth from CHL or data
```

File: content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60355
+version: 1020.16.1
+---
+otpAuth from chl or data (#6869) [GRAFT][release/cd] (#6883)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md:17]: The Claude AI suggested to use the following body:
```suggestion
To enhance the flexibility of the authentication process, support for the otpAuth parameter has been added. This parameter can now be provided either via the CHL interface or directly through the request data. With this change, users have more options to include the otpAuth value when authenticating, depending on their specific setup and requirements. Existing authentication flows will continue to work as before, while integrations that rely on passing otpAuth via CHL or data can now leverage this capability.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md (hunk 1 above)
Comment (kgol-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md:3]:
```suggestion
title: QR code for TOTP is now shown correctly 
```

File: content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md (hunk 2 above)
Comment (kgol-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md:17]: QR code for TOTP two-factor authentication was not showing correctly. Now the fix has been made to show QR code correctly for TOTP

Comment (kgol-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md:17]:
```suggestion
QR code for TOTP two-factor authentication was not showing correctly. Now the fix has been made to show QR code correctly for TOTP
```

**Review by kgol-c8y — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60355
+version: 1020.16.1
+---
+The QR code for the TOTP two-factor authentication was not displayed correctly. This issue has been fixed and now the QR code for TOTP shows up correctly.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md:17]:
```suggestion
The QR code for the two-factor authentication provided during the TOTP setup was not displayed correctly. This issue has been fixed. The QR code is now properly visible and users can scan it with their authentication application.
```

File: content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: QR code for TOTP is now shown correctly
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-16-1-otp-auth-from-chl-or-data-6869-graft-release-cd-6883.md:3]:
```suggestion
title: QR code for two-factor authentication provided during TOTP setup now shows up correctly 
```

**Pull request merged**